docs/src/inverse_problems/petab_ode_param_fitting.md

```diff
@@ -0,0 +1,487 @@
+```petab1
+@unpack S = rn
+event1 = PEtabEvent(5.0, S + 0.5, S)
+```
+Here, the first argument is evaluated to a scalar value, in which case it is interpreted as a time point at which the event happens. If we instead want the event to happen whenever the concentration of `S` falls below `0.5` we set the first argument to a boolean condition indicating this:
+```petab1
+event2 = PEtabEvent(S < 0.5, S + 0.5, S)
+```
+
+Whenever we have several events or not, we bundle them together in a single vector which is later passed to the `PEtabODEProblem` using the `events` argument:
+```petab1
+events = [event, event2]
+petab_model = PEtabModel(rn, observables, measurements, params; state_map=u0, events=events)
+```
+
+More details on how to use events, including how to create events with multiple targets, can be found in [PEtab.jl's documentation](https://sebapersson.github.io/PEtab.jl/stable/Julia_event/).
+
+## [Plot recipes](@id petab_plotting)
+There exist various types of graphs that can be used to evaluate the parameter fitting process. These can be plotted using the `plot` command, where the input is either the result of a `calibrate_model` or a  `calibrate_model_multistart` run. To be able to use this functionality, you have to ensure that PEtab.jl [records the optimisation process](@id petab_optimisation_path_recording) by providing the `save_trace=true` argument to the calibration functions.
+
+To, for a single start calibration run, plot, for each iteration of the optimization process, the best objective value achieved so far, run:
+
+```petab1
+plot(res)
+```
+For a multi-start calibration run, the default output is instead a so-called waterfall plot:
+```petab1
+plot(res_ms)
+```
+![petab waterfall plot](../assets/petab_waterfall.svg)
+(for this, and the next plot, we use a multi-start optimisation result from a different model, which yields less trivial optimisation runs than our catalysis one)
+
+In the waterfall plot, each dot shows the final objective value for a single run in the multi-start process. The runs are ordered by their objective values, and colours designate runs in the same local minimum.
+
+To instead use the best objective value plot for a multi-start run (with one curve for each run), the `plot_type` argument is used:
+```petab1
+plot(res_ms; plot_type = :best_objective)
+```
+![petab best objective plot](../assets/petab_best_objective.svg)
+
+There exist several types of plots for both types of calibration results. More details of the types of available plots, and how to customise them, can be found [here](@ref https://sebapersson.github.io/PEtab.jl/stable/optimisation_output_plotting/).
+
+
+## [Citations](@id petab_citations)
+If you use this functionality in your research, [in addition to Catalyst](@ref catalyst_citation), please cite the following papers to support the authors of the PEtab.jl package (currently there is no article associated with this package) and the PEtab standard:
```

Comment:
If the paper ever escapes the review land we can add it here :)

Reply:
Definitely :)